src/components/modals/ErrorModal.vue

```diff
@@ -28,6 +28,14 @@ function close() {
                     <h5 class="title is-5">Suggestion</h5>
                     <p>{{solution}}</p>
                 </div>
+                <div class="mt-3 text-right" v-if="error.extraAction">
+                    <button 
+                        class="button error-action-button is-danger"
```

Comment:
Would `is-white` look better than `is-danger`? Then we wouldn't need the CSS rule for border either.